market/demos/excel-connector/meta.json

```diff
@@ -0,0 +1,73 @@
+{
+	"id": "excel-connector-demo",
+	"version": "9.3",
+	"name": "Excel Connector Demo",
+	"description": "Simple demo that shows you how to import/export Ivy Recordsets using Excel.",
+	"type": "solution",
+	"vendor": "Axon Ivy AG",
+	"platformReview": "4.5",
+	"sourceUrl": "https://github.com/axonivy-market/excel-connector",
+	"statusBadgeUrl": "https://github.com/axonivy-market/excel-connector/actions/workflows/ci.yml/badge.svg",
+	"language": "English",
+	"industry": "Cross-Industry",
+	"compatibility": "8.0+",
+	"tags": [
+		"demo"
```

Comment:
no demo: maybe a 'util'? ... but surely our PM will later contribute the final tags.